karyon3-admin-simple/src/main/java/com/netflix/karyon/admin/HttpServerModule.java

```diff
@@ -0,0 +1,21 @@
+package com.netflix.karyon.admin;
+
+
+public final class HttpServerModule extends AbstractAdminModule {
+    @Override
+    protected void configure() {
+        bind(SimpleHttpServerStarter.class).asEagerSingleton();
+        bind(HttpServerRegistry.class).to(GuiceHttpServerRegistry.class);
+        bindAdminResource("http").to(HttpServerAdminResource.class);
+    }
+    
+    @Override
+    public boolean equals(Object obj) {
+        return HttpServerModule.class.equals(obj.getClass());
```

Comment:
Why not just ```getClass().equals(obj.getClass())```?

nit: The contract for equals says you should check for obj being null to avoid an NPE.